gson/src/test/java/com/google/gson/JsonPrimitiveTest.java

```diff
@@ -297,7 +297,7 @@ public void testEqualsAcrossTypes() {
         new JsonPrimitive(new BigInteger("0")), new JsonPrimitive(0));
     MoreAsserts.assertEqualsAndHashCode(new JsonPrimitive(0), new JsonPrimitive(0L));
     MoreAsserts.assertEqualsAndHashCode(
-        new JsonPrimitive(new BigInteger("0")), new JsonPrimitive(0));
+        new JsonPrimitive(new BigDecimal("0")), new JsonPrimitive(0));
```

Comment:
Changed this because the same test (`new BigInteger("0"), new JsonPrimitive(0)`) already exists three lines above.